negotiation/neural/model_builder.py

```diff
@@ -149,9 +149,8 @@ def make_decoder(opt, embeddings):
                              embeddings=embeddings)
 
 
-def load_test_model(opt, dummy_opt):
-    checkpoint = torch.load(opt.model,
-                            map_location=lambda storage, loc: storage)
+def load_test_model(model_path, uses_gpu, dummy_opt):
+    checkpoint = torch.load(model_path, map_location=lambda storage, loc: storage)
```

Comment:
why did you delete opt? there are some options that we want to specify at run time instead of reading from the model checkpoint